tests/test_handler.py

```diff
@@ -223,6 +225,90 @@ def test_exception_handler_on_web_request(self):
         self.assertEqual(response["statusCode"], 500)
         mocked_exception_handler.assert_called()
 
+    def test_wsgi_script_binary_support_base64_behavior(self):
```

Comment:
Could this be broken into a unit test per relevant scenario to make it more clear what is the expected behavior in each scenario?  Eg. one for `application/json`, one for `text/*`, one for `foo/bar`, etc.

Reply:
I think that's a great idea. I was trying to match the style of the tests when I wrote them, but I do think that style feels claustrophobic. Will do.

Reply:
Tests have been divided up into three test methods based on what they're checking for:
- `test_wsgi_script_binary_support_with_content_encoding`
- `test_wsgi_script_binary_support_without_content_encoding_edgecases` (where the mimetype is inspected)
- `test_wsgi_script_binary_support_without_content_encoding`